src/test/java/com/artipie/pypi/PypiITCase.java

```diff
@@ -124,19 +158,16 @@ private String exec(final String... command) throws Exception {
         return this.cntn.execInContainer(command).getStdout();
     }
 
-    private String config() {
-        return Yaml.createYamlMappingBuilder().add(
-            "repo",
-            Yaml.createYamlMappingBuilder()
-                .add("type", "pypi")
-                .add(
-                    "storage",
-                    Yaml.createYamlMappingBuilder()
-                        .add("type", "fs")
-                        .add("path", this.tmp.resolve("repos").toString())
-                        .build()
-                )
-                .build()
-        ).build().toString();
+    private void prepareDirectory(final String dist) throws IOException {
+        FileUtils.copyDirectory(
```

Comment:
@genryxy I'd suggest to perform this operation in our "storage terms": `TestResource` has `addFilesTo()` method to copy resources to provided storage and base key. Also, as this will be just one expression, I think that method can be inlined